CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+# Contributing to @ekino/logger 🌟
+
+First and foremost, thank you for considering a contribution to this project! 💖 
+
+Your time and effort make a difference, and every pull request, no matter the size, helps move this project forward.
+
+Whether you're fixing a bug, adding a new feature, or improving documentation, your input is valuable. If you're new to open source, welcome! 🎉 
+
+Don't worry if you feel unsure about where to start—every bit of help is appreciated, and questions are welcome.
+
+## How to Contribute
+
+Follow this guide to ensure a smooth contribution process. If you're planning significant work or substantial changes, please create an issue labeled "contribution enquiry" to discuss your proposal. This will help ensure that your work aligns with the project's goals.
```

Comment:
1. The label does not exist
2. A new contributor cannot add labels, most of the time a member must set the labels for him